irispy/iris_tools.py

```diff
@@ -85,13 +90,17 @@ def calculate_intensity_fractional_uncertainty(data, data_unit, detector_type):
         Same shape as data.
 
     """
+    if 'FUV' in detector_type:
+        detector_type = 'FUV'
+
     photons_per_dn = DETECTOR_GAIN[detector_type]/DETECTOR_YIELD[detector_type]
     if data_unit == "DN":
         intensity_ph = photons_per_dn*convert_DN_to_photons(data, detector_type)
```

Comment:
@DanRyanIrish is this line correct should it be multiplied by photons_per_dn after it has been converted to photons